Rakefile

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/rake
+require 'yaml'
+require 'json'
+require 'net/http'
+require 'uri'
+require 'plist'
+
+desc 'Bootstrap this repository for development'
+task :bootstrap do
+  [:StencilSwiftKit, :SwiftGenKit, :SwiftGen, :templates].each do |repository|
+    next if Dir.exists? "#{repository}"
+
+    sh "git clone git@github.com:SwiftGen/#{repository}.git --recursive"
+    Dir.chdir("#{repository}") do
+      sh "git submodule update --init --recursive"
+    end
+  end
+end
```

Comment:
What I often do when working on a repo with a submodule, and create changes in both repos, I just commit from within the submodule. That way, the submodule reference is updated in the parent repo when I commit in there.

When creating releases for one of the kits or swiftgen itself, a script to update all of them would be useful.